[20170] Order the options in the modal selects within the WP List

floriank opened the pull request:
This will order the select options alphabetically, as requested in https://community.openproject.org/work_packages/20170.

Commit:
order the options in the modal selects

this will order the select options alphabetically

Signed-off-by: Florian Kraft <f.kraft@finn.de>

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://floriank-fix-20170-sorting-in-wp-list-menus.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://floriank-fix-20170-sorting-in-wp-list-menus.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

Commit:
edit the cukes to reflect a new reality

These cukes broke as the button for the new subproject was moved around in
#3136.

Signed-off-by: Florian Kraft <f.kraft@finn.de>

ulferts committed:
Merge pull request #3145 from floriank/fix/failing-cukes-in-project-create

[18714] Repair broken cukes for creating a new subproject

floriank commented:
Wrong branch.
